content/docs/learning-guide/tutorials/environments/constructing-terrain.md

```diff
@@ -0,0 +1,96 @@
+---
+linkTitle: Constructing Terrain
+title: Construct terrain in Open 3D Engine
+description: Learn to construct terrain in Open 3D Engine (O3DE).
+weight: 200
+toc: true
+---
+
+## **Constructing Terrain**
+You can construct terrain via the Entity Outliner and Entity Inspector using gradient generators.
+
+**Setup**: Enable the Terrain Gem.
+
+*Note: You will need to create both a spawner area and spawnable area.*
+
+### **Define a spawnable area**
+
+1. In the **O3DE Editor**, create a new level or open an existing level.
+2. In the **Entity Outliner**, select the level.prefab.
+- ![A](/images/learning-guide/tutorials/environments/constructing-terrain/A.png)
+3. In the **Entity Inspector**, add both the **Terrain World** and **Terrain World Renderer** components.
+- ![B](/images/learning-guide/tutorials/environments/constructing-terrain/B.png)
+
+### **Define a spawnable area**
+
+1. Create a new entity (hotkey **Ctrl+Alt+N**). (This will be your **TerrainSpawner** entity.)
+- ![C](/images/learning-guide/tutorials/environments/constructing-terrain/C.png)
+2. With the new entity selected, in the Entity Inspector, add a **Terrain Layer Spawner** and an **Axis Aligned Box Shape** component. (You can adjust the dimensions of the Aligned Box Shape component to encompass or intersect with the Terrain Layer Spawner.)
```

Comment:
These steps should also mention the following:
* The position of the entity represents the center of the Axis Aligned Box Shape, so a box of 100 x 100 x 100 m will cover an area 50 m in each direction from the entity position.
* The box should be positioned within the world height bounds that were set on the Terrain World component. The min/max heights on that component are 0 to 1024 m by default, so the entity should be positioned high enough that the bottom of the box is at least 0. (Or alternatively, the min/max heights can be adjusted to go below 0)